### ISSUE TITLE
Canned polls

### ISSUE THREAD

**Issue by R-0-Y** [strawpoll/strawpoll#13]
Hi,
I think about a little features that could be nice, sometimes I see some streamer doing the same poll at every stream 
eg: in Fasther Than Light, wich ship to use ? I see a streamer put the 28 ship one by one every time he stream FTL.
 An option to save a poll question/answer and use him will be cool !

**Comment by dandirks** [comment]
This is a great suggestion. I will look into the ability to have "canned polls" where a user can store a particular question with options that will allow them to create a duplicate of that poll whenever they so choose.

**Comment by dandirks** [comment]
I have completed this issue on my dev instance. It has been implemented as a "Save as draft" link when creating a new poll, as seen below:

![savedraft](https://cloud.githubusercontent.com/assets/552824/8489809/8e041d06-20d4-11e5-844c-75765561497b.png)

After clicking this link, the URL of the page is updated to include a serialized version of the information that has been entered. This is then used to restore that information whenever this URL is visited again. In the example you outlined, the streamer that you watch can enter the title and options he wants for his FTL poll, click the "Save as draft" link, and then bookmark the URL to come back to later whenever he wants to make a duplicate of the poll.

This feature will be part of the next release of Straw Poll. I am closing this issue for now to better reflect what still needs to be done.
